dashboard/src/components/Config/AppRepoList/AppRepoList.tsx

```diff
@@ -113,15 +117,18 @@ class AppRepoList extends React.Component<IAppRepoListProps> {
           namespace={namespace}
         />
         {displayReposPerNamespaceMsg && (
-          <MessageAlert header="Looking for site-wide app repositories?">
+          <MessageAlert header="Looking for other app repositories?">
             <div>
               <p className="margin-v-normal">
-                You can view site-wide App Repositories by selecting "All Namespaces" above, if you
-                have permission to view or edit App Repositories available to all namespaces.
+                You can view App Repositories across all namespaces by selecting "All Namespaces"
+                above, if you have permission to view App Repositories cluster-wide. We will also
+                add information on the current page indicating App Repositories in other namespaces
+                which can be used in the current namespace.
```

Comment:
The second sentence is a bit confusing. I am not sure what it means?

Reply:
I was trying to say something which you were also trying to say on a different thread: that we will, later on, update this view to include displaying other app repositories which have been made available in this namespace, even though they are not defined here. I agree it's quite confusing and is an implementation detail which we don't need to expose to the user. For now I'm just removing this sentence and we'll later include those.